pkg/storegateway/bucket_stores_test.go

```diff
@@ -513,8 +583,11 @@ func queryLabelsNames(stores *BucketStores, userID, metricName string) (*storepb
 	return stores.LabelNames(ctx, req)
 }
 
-func queryLabelsValues(stores *BucketStores, userID, metricName string) (*storepb.LabelValuesResponse, error) {
+func queryLabelsValues(stores *BucketStores, userID, metricName string, start, end int64) (*storepb.LabelValuesResponse, error) {
 	req := &storepb.LabelValuesRequest{
+		Start: start,
+		End:   end,
+		Label: "__name__",
```

Comment:
Let's also make this part of the func signature?

Reply:
Yeah! lets do that!